src/Form/FieldInput.js

```diff
@@ -222,6 +233,8 @@ FieldInput.propTypes = {
   handleEvent: React.PropTypes.func,
   // Optional label to add
   label: React.PropTypes.string,
+  // Optional hepl block
+  helpBlock: React.PropTypes.node,
```

Comment:
Ahh cool, I didn't realize that. Should we call `classnames` on it as we do in [here](https://github.com/mesosphere/reactjs-components/blob/dfea31110f87fe945419749fefde3fa84e2f08c7/src/Form/FieldInput.js#L94)?